auth/realm/base/src/main/java/org/wildfly/security/auth/realm/ElytronMessages.java

```diff
@@ -130,6 +130,10 @@
     @Message(id = 1154, value = "Failed to read key store")
     RealmUnavailableException failedToReadKeyStore(@Cause KeyStoreException e);
 
+    @Message(id = 13005, value = "Filesystem-backed realm encountered invalid SecretKey")
```

Comment:
It's better to add new messages to the bottom of the file so we can double check the id.

Reply:
s/invalid/an invalid

Reply:
Fixed